templates/components/cart/content.html

```diff
@@ -19,13 +19,17 @@
                             title="{{lang 'cart.gift_certificate'}}"
                         >
                     {{else}}
-                        {{> components/common/responsive-img
-                            image=image
+                        <img
+                            {{#if image}}
+                                src="{{getImageSrcset image 1x=(default fallback_size ../theme_settings.productthumb_size)}}" 
+                                srcset="{{getImageSrcset image 1x=(default fallback_size '200x200')}} 2x"
+                                title="{{image.alt}}"
+                                alt="{{image.alt}}"    
+                            {{else if default_image}}
+                                src="{{cdn default_image}}
+                            {{/if}}
```

Comment:
Can you check this please: 
When _responsive-img component_ has been used we explicitly provide _default_image_ path `../theme_settings.default_image_product` there (in old version Line27) since it’s not within _cart.items_ array. If we remove the component then mentioned condition never works.

Am I wrong or we need rewrite/remove condition?

Reply:
Yes that is correct - ../theme_settings.default_image_product value should be instead of variable here. I just forgot to push this update - its already done. But thanks for attention:)